{"name": "test_another", "status": "skipped", "statusDetails": {"message": "Skipped: [ISSUE-23444] Issue is network connection", "trace": "('/home/ivan/PycharmProjects/pythonProject/solveme_pytest/tests/users/test_users.py', 13, 'Skipped: [ISSUE-23444] Issue is network connection')"}, "start": 1659718527975, "stop": 1659718527975, "uuid": "56dd5fdc-05a7-4510-bc40-91eb962815a0", "historyId": "ecdad97ca3d7a3be9fe1de0e65fa7b4f", "testCaseId": "19630dc244333b530cc14c893795ee5f", "fullName": "tests.users.test_users#test_another", "labels": [{"name": "tag", "value": "@pytest.mark.skip('[ISSUE-23444] Issue is network connection')"}, {"name": "tag", "value": "production"}, {"name": "tag", "value": "development"}, {"name": "parentSuite", "value": "tests.users"}, {"name": "suite", "value": "test_users"}, {"name": "host", "value": "ivan-pc"}, {"name": "thread", "value": "52803-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.users.test_users"}]}